examples/darwin-framework-tool/commands/common/CHIPCommandBridge.mm

```diff
@@ -76,15 +102,20 @@
         mControllers[identities[i]] = controller;
     }
 
-    // If no commissioner name passed in, default to alpha.
-    SetIdentity(mCommissionerName.HasValue() ? mCommissionerName.Value() : kIdentityAlpha);
-
-    ReturnLogErrorOnFailure(RunCommand());
-    ReturnLogErrorOnFailure(StartWaiting(GetWaitDuration()));
-
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR CHIPCommandBridge::MaybeTearDownStack()
+{
+    CHIP_ERROR err;
+    if (IsInteractive()) {
+        return CHIP_NO_ERROR;
+    }
+    err = ShutdownCommissioner();
```

Comment:
What does the shutdown in interactive mode?

Reply:
Can you please rephrase? I having trouble understanding this question.

Reply:
What calls ShutdownCommissioner() in interactive mode?  Does the "quit()" command handling do that?

Reply:
Yes, the "quit()" command is the only thing at the moment that calls `ShutdownCommissioner`. We may need to add signal handlers to catch and shutdown the commissioners. Should I file a followup?

Reply:
No, if quit() does it we're good here.